README.md

```diff
@@ -40,50 +47,6 @@ An example for a "porcelain" API would be [ROS2](https://www.ros.org/). Others a
 | [RTA-VRTE](https://www.etas.com/en/products/rta-vrte.php) | [Adaptive AUTOSAR](https://www.autosar.org/standards/adaptive-platform/) platform software framework for vehicle computer from [ETAS GmbH](https://www.etas.com) |
 | [Cyclone DDS](https://github.com/eclipse-cyclonedds/cyclonedds) | Performant and robust open-source DDS implementation maintained by [ADLINK Technology Inc.](https://www.adlinktech.com/) |
 
-### Targeted quality levels & platforms
-
-> [Quality level](./CONTRIBUTING.md#quality-levels) are 5..1, where 1 is highest
-
-|CMake project/target   | QNX  | Linux, Windows, MacOS | Comment                             |
-|-----------------------|:----:|:---------------------:|:-----------------------------------:|
-| example_icedelivery   | 5    | 5                     |                                     |
-| example_iceperf       | 5    | 5                     |                                     |
-| example_singleprocess | 5    | 5                     |                                     |
-| iceoryx_dds           | 4    | 4                     |                                     |
-| iceoryx_meta          | 5    | 5                     |                                     |
-| iceoryx_posh          | 1, 2 | 4                     | Will be split into separate targets |
-| iceoryx_utils         | 1    | 4                     |                                     |
-| iceoryx_introspection | 5    | 5                     |                                     |
-
-### Scope
-
-Who can benefit of using iceoryx? What's in for those folks?
-
-#### User personas
-
-**Andrew, the HAD developer**
```

Comment:
Three personas are maybe too much text, but can we maybe have the HAD developer? From my point of view it adresses quite good the target group.

Reply:
I wanted to reduce the content of the main README. Maybe we could have them in another file if you have one in mind?

Reply:
I don't know where to put this, but I think we should also state our goals and non-goals (http://www.cleverpm.com/2018/01/11/goals-non-goals/) and this could be placed in the same file with the personas

Reply:
Maybe we can also move the build and install instructions to a separate file and just link from the README to this file.
The README is currently really to big and it might scare people away who just want to have a quick look what iceoryx does.

Reply:
like your ideas @elBoberido, even if it makes more work

Reply:
@elBoberido Please check again now